src/app/exercise/exercise.page/components/exercise-toasters.directive.ts

```diff
@@ -74,12 +74,12 @@ export class ExerciseToastersDirective {
           toaster.present();
         });
 
-      return () => {
+      onCleanup(() => {
         if (lastToaster) {
           lastToaster.dismiss();
           lastToaster = null;
         }
-      };
+      });
```

Comment:
I think Angular changed the way cleanup works with effects, or I initially misunderstood when migrating to signals.